config/config/sc-config.yaml

```diff
@@ -760,3 +760,10 @@ s3Exporter:
   tolerations: []
   affinity: {}
   nodeSelector: {}
+prometheusBlackboxExporter:
+  targets:
+    rook: false
+    nginx: false
+    fluentd: false
+    thanosQuery: false
+    prometheus: false
```

Comment:
Since these are installed by default I think we should have them enabled by default as well